data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,271 @@ mission "Arachi Cartoonist 2"
+				accept
+			label shrine
+			`	She takes a long, deep breath. "Thank you, Captain. Far outside the village, the site is. Come, show you the way there, I will." You leave your ship and follow her to the railway hub and board one of trains that appears to be one of the oldest and most simple. It lacks seats for non-Saryds, so you are forced to hold tight to what handrails you can reach.`
+			`	You remain on the train until the very last stop, when the sun is already starting to set. As you depart the train, you see a rural village with just a few dozen buildings and a simple dirt road, where a couple workers battle the snow and ice to keep it clear and easy to traverse. You follow Vellorae down the road, and once you reach a rocky pillar with a hole carved through it, Vellorae turns to the forest, heading out on a thin, straight trail. Looking behind you, you see the setting sun fitting inside the pillar's opening.`
+			`	As you follow Vellorae down the trail, you see the trees gradually show more and more signs of a forest fire, with darker, damaged barks, increasingly ashy soil, and many fallen, decomposing portions of the forest. The trail ends abruptly at a large scar scorched through the forest. The piles of ash still present below your feet almost resemble the rest of the planet's otherwise snowy ground, but somehow are kept from mixing with the snow. Whether deliberately cleared by more workers or some other force, you cannot tell. Near the center of the scar you see a small mound with an opening in the side, almost like a "shack" made out of the charred tree trunks and rocks. A faint light comes from within it.`
+			`	While you're looking at the mound, Vellorae grabs your hand and starts hesitantly pulling you towards it. Inside, the fading sunlight behind you contrasts with the source of the light within, a small pyre in the middle of the "shack" which has only a few lone flames dancing on the embers. Vellorae lays down in front of it, and removes her bangle and places it in the flames for a brief moment before snatching it up and standing again. Once on her feet, she grabs your hand again, tugging you along as she hurries back to the edge of the scorched clearing.`
+			`	She seems to settle again once you have reached the trail, letting go of your hand and slowing her pace.`
+			choice
+				`	"What was that about? What got you so agitated?"`
+				`	(Stay quiet.)`
+					goto return
+			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`
```

Comment:
Would this be ok?
```suggestion
			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but completely destroyed, it was not. To implore Adfices is kept safe from it, this ritual was for."`
```

Reply:
Personally I prefer EjoThims' suggestion, save for what I commented about it. Just waiting for it to be updated.

Reply:
Alright then.